contrib/opencensus-ext-azure/opencensus/ext/azure/events_exporter/__init__.py

```diff
@@ -0,0 +1,24 @@
+from opencensus.ext.azure.common import utils
+from opencensus.ext.azure.common.protocol import Data, Envelope, Event
+from opencensus.ext.azure.log_exporter import AzureLogHandler
+
+
+class AzureEventHandler(AzureLogHandler):
```

Comment:
I don't think we need another handler type if we are going to be using `Message` telemetry type. We can leverage the already existing `AzureLogHandler` and place the new "properties" logic in there.